src/DependencyInjection/Configuration.php

```diff
@@ -69,6 +69,12 @@ public function getConfigTreeBuilder()
                         ->scalarNode('server')->cannotBeEmpty()->end()
                         ->scalarNode('enabled')->defaultFalse()->end()
                         ->arrayNode('ip_white_list')
+                            ->prototype('scalar')->end()
+                            ->defaultValue(['127.0.0.1'])
+                            ->info('IPs for which 2FA will be skipped. DEPRECATED: replace by "trusted_ip_list" instead parameter with the same values')
+                            ->setDeprecated()
```

Comment:
I'm confused… why is `setDeprecated` empty now?

Reply:
Fixed